src/Http/Controllers/LibraryApiController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace EscolaLms\HeadlessH5P\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use EscolaLms\HeadlessH5P\Http\Controllers\Swagger\LibraryApiSwagger;
+use EscolaLms\HeadlessH5P\Services\HeadlessH5PService;
+use EscolaLms\HeadlessH5P\Http\Requests\LibraryStoreRequest;
+
+class LibraryApiController extends Controller implements LibraryApiSwagger
+{
+    private HeadlessH5PService $hh5pService;
```

Comment:
Again shouldn't that be `$h5pService` insteed of `$hh5pService`

Reply:
And all below

Reply:
`hh5pService` headless h5p service